blocks/HARDWARE/OSCILLOSCOPES/TEKTRONIX/MSO2X/DIGITAL_CHANNELS_MSO2X/DIGITAL_CHANNELS_MSO2X.py

```diff
@@ -0,0 +1,103 @@
+from typing import Optional, Literal
+from flojoy import VisaConnection, flojoy, DataContainer, String
+
+
+@flojoy(deps={"tm_devices": "1"}, inject_connection=True)
+def DIGITAL_CHANNELS_MSO2X(
+    connection: VisaConnection,
+    input: Optional[DataContainer] = None,
+    all_channels: bool = False,
+    D0_display: Literal["1", "0"] = "0",
+    D1_display: Literal["1", "0"] = "0",
+    D2_display: Literal["1", "0"] = "0",
+    D3_display: Literal["1", "0"] = "0",
+    D4_display: Literal["1", "0"] = "0",
+    D5_display: Literal["1", "0"] = "0",
+    D6_display: Literal["1", "0"] = "0",
+    D7_display: Literal["1", "0"] = "0",
+    D8_display: Literal["1", "0"] = "0",
+    D9_display: Literal["1", "0"] = "0",
+    D10_display: Literal["1", "0"] = "0",
+    D11_display: Literal["1", "0"] = "0",
+    D12_display: Literal["1", "0"] = "0",
+    D13_display: Literal["1", "0"] = "0",
+    D14_display: Literal["1", "0"] = "0",
+    D15_display: Literal["1", "0"] = "0",
```

Comment:
May make more sense to just make all of these booleans and convert to 1/0 in the function, I think checkboxes make more sense from a user's perspective